pub-sub/README.md

```diff
@@ -89,63 +89,63 @@ python -m pip install -r requirements.txt
 dapr run --app-id python-subscriber --app-port 5000 python3 app.py
 ```
 
-### Use the CLI to publish messages to subscribers
+### Run the React front end with Dapr
 
-The Dapr CLI provides a mechanism to publish messages for testing purposes. Now, test that subscribers are listening!
+Now, run the React front end with Dapr. The front end will publish different kinds of messages that subscribers will pick up.
 
-1. Use Dapr CLI to publish a message:
-   
-  * Linux/Mac:
+1. Open a new CLI window and navigate to the react-form directory:
 
 ```bash
-dapr publish --topic A --pubsub pubsub --data '{ "message": "This is a test" }'
+cd react-form
 ```
-    
-  * Windows CMD
+
+2. Run the React front end app with Dapr: 
 
 ```bash
-dapr publish --topic A --pubsub pubsub --data "{ \"message\": \"This is a test\" }"
+dapr run --app-id react-form --app-port 8080 npm run buildandstart
 ```
-    
-  * Windows Powershell
 
-```ps
-dapr publish --topic A --pubsub pubsub --data "{ message: This is a test }"
-```
+This may take a minute, as it downloads dependencies and creates an optimized production build. You'll know that it's done when you see `== APP == Listening on port 8080!` and several Dapr logs.
 
-Both Node.js and Python subscribers subscribe to topic A and log when they receive a message. Note that logs are showing up in the console window where you ran each one: 
+3. Open the browser and navigate to "http://localhost:8080/". You should see a form with a dropdown for message type and message text: 
+
+![Form Screenshot](./img/Form_Screenshot.JPG)
+
+4. Pick a topic, enter some text and fire off a message! Observe the logs coming through your respective Dapr. Note that the Node.js subscriber receives messages of type "A" and "B", while the Python subscriber receives messages of type "A" and "C". Note that logs are showing up in the console window where you ran each one: 
 
 ```bash
 [0m?[94;1m== APP == Topic A: { id: '5780e2ca-f526-4839-92e5-a0a30aff829a', source: 'react-form', type: 'com.dapr.event.sent', specversion: '0.3',data: { message: 'this is a test' } }
 ```
 
-2. **Optional**: If you're running both the Node and Python apps, try publishing a message of topic B. You'll notice that only the Node app will receive this message. A more detailed discussion on how these microservices are subscribing to specific topics can be found in [How it Works](#How-it-Works).
-
-### Run the React front end with Dapr
+### Use the CLI to publish messages to subscribers
 
-Now, run the React front end with Dapr. The front end will publish different kinds of messages that subscribers will pick up.
+The Dapr CLI provides a mechanism to publish messages for testing purposes.
 
-1. Open a new CLI window and navigate to the react-form directory:
+1. Use Dapr CLI to publish a message:
+   
+  * Linux/Mac:
 
 ```bash
-cd react-form
-```
 
-2. Run the React front end app with Dapr: 
+dapr publish --topic A --pubsub react-form --data '{ "message": "This is a test" }'
```

Comment:
i think pubsub should be the same `pubsub` and the publish-app-id should be `react-form`

Reply:
My mistake.

Reply:
Fixed.